src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -442,10 +489,46 @@ def _enable_module_qat(self, module: Module):
         # set modules with proper qconfigs to QAT mode
         torch_quantization.prepare_qat(module, inplace=True)
         if self._quantize_embeddings:
-            prepare_embeddings_qat(module, reduce_range=self._reduce_range)
+            prepare_embeddings_qat(
+                module,
+                reduce_range=self._reduce_range,
+                activation_qconfig_kwargs=activation_qconfig_kwargs,
+                weight_qconfig_kwargs=self.weight_qconfig_kwargs,
+            )
+
+        # propagate custom quant min/max range from FakeQuantize to Observer objects
+        fix_observer_quant_range(module)
 
         self._qat_enabled = True
 
+    def _get_updated_activation_qconfig_kwargs(self):
+        # update qconfig_kwargs
+        if self.activation_bits and (
+            self.activation_qconfig_kwargs.get("quant_min")
+            or self.activation_qconfig_kwargs.get("quant_max")
+        ):
+            raise ValueError(
+                "Cannot override quant_max and quant_min with activation_bits enabled"
+            )
+
+        activation_qconfig_kwargs = self.activation_qconfig_kwargs.copy()
+        quant_min = activation_qconfig_kwargs.get("quant_min", 0)
+        quant_max = activation_qconfig_kwargs.get("quant_max", 2 ** 8 - 1)
+        dtype = activation_qconfig_kwargs.get("dtype", torch.quint8)
```

Comment:
no need to set a default here when a user does not override activaiton_bits or the kwargs, that will be handled later in our pipeline. let's delete this block and move the dict update inside the if statement